src/test/java/org/sagebionetworks/web/unitclient/presenter/LoginPresenterTest.java

```diff
@@ -108,10 +115,6 @@ public void setup() {
       .callSuccessWith(mockUserProfile)
       .when(mockAuthenticationController)
       .setNewAccessToken(anyString(), any(AsyncCallback.class));
-    AsyncMockStubber
-      .callSuccessWith(mockUserProfile)
-      .when(mockAuthenticationController)
-      .initializeFromExistingAccessTokenCookie(any(AsyncCallback.class));
```

Comment:
The `AsyncMockStubber` util doesn't like when a method has extra params other than callbacks